backend/server/static/api/meetingbook.yaml

```
@@ -163,22 +164,230 @@ paths:
         "401":
           description: "Unauthorized"
       security:
-        - basic_auth: []    
-  /bookings/{booking_id}:
+        - basic_auth: []
+  /booking_settings:
+    summary: endpoint for the settings of the booking, controlled by admin.
+    get:
+      summary: get current settings
+      parameters:
```

```suggestion
```

---

or need to write parameters

---

We need to add parameters:

```suggestion
      parameters:
      - in: query
      name: duration
      type: string
      required: true
      - in: query
      name: start_time
      type: string
      required: true
```

---

Ack